web/src/app/tree/shared/tree-diagram.service.ts

```diff
@@ -71,6 +108,11 @@ function init() {
           // @ts-ignore
           $(GenogramLayout, { direction: 90, layerSpacing: 30, columnSpacing: 10 })
       });
+      if (title) {
+        myDiagram.add(
+          $(go.Part, { location: new go.Point(0, -40) },
+            $(go.TextBlock, title, { font: `bold ${titleSize ? titleSize : '24'}pt sans-serif`, stroke: "black" })));
+      }
```

Comment:
The title is a pickupable graph element by default?

Reply:
Yes